src/config/enhance-octokit.ts

```diff
@@ -1,76 +0,0 @@
-import { statsd }  from "./statsd";
-import { GitHubAPI } from "probot";
-import { metricError, metricHttpRequest } from "./metric-names";
-import { getLogger } from "./logger";
-import { Octokit } from "@octokit/rest";
-import { extractPath } from "../jira/client/axios";
-
-const logger = getLogger("octokit");
-
-export class RateLimitingError extends Error {
-	/**
-	 * The value of the x-ratelimit-reset header, i.e. the epoch seconds when the rate limit is refreshed.
-	 */
-	rateLimitReset: number;
-
-	constructor(resetEpochSeconds: number) {
-		super("rate limiting error");
-		this.rateLimitReset = resetEpochSeconds;
-		Object.setPrototypeOf(this, RateLimitingError.prototype);
-	}
-}
-
-const instrumentRequests = (octokit: GitHubAPI) => {
```

Comment:
So is it:
1. In this PR, that feature of `instrumentRequests` are lost in code?
2. And this feature in prod, is actually already lost due to FF turned on?

Reply:
on friday we removed about 6 feature flags, the code no longer uses this(hasn't for over a month with flags on). It is run in prod but never accessed since we use the GitHub clients instead of Octokits offering

Reply:
Never mind, the instrument is added somewhere else already. All good 👍